utils/python/CIME/XML/entry_id.py

```diff
@@ -15,92 +15,66 @@ def __init__(self, infile=None):
         GenericXML.__init__(self, infile)
         self.groups={}
 
-    def set_default_value(self, vid, attributes=None):
+    def set_default_value(self, node, attributes=None):
         """
         Set the value of an entry to the default value for that entry
-        vid can be an xml node pointer or a string identifier of a node
         """
-        value = None
-        if (type(vid) != type(str())):
-            node = vid
-            vid = node.attrib["id"]
-        else:
-            nodes = self.get_node("entry", {"id":vid})
-            if (nodes is None):
-                return
-            expect(len(nodes) == 1, "More than one match found for id " + vid)
-            node = nodes[0]
-
-        valnodes = self.get_node("value",root=node)
-        if (valnodes is not None):
-            for valnode in valnodes:
-                for att in valnode.attributes:
-                    if (att.key in attributes):
-                        if (re.search(attributes[att.key],att.text)):
-                            value = valnode.text
-                            logger.info("id %s value %s" % (vid, valnode.text))
-
-        if (value is None):
-            value = self.get_node("default_value", root=node)
-        if (value is not None):
-            node.set("value", value[0].text)
-            return value[0].text
-
-    def set_value(self, vid, value,subgroup=None):
+        # Need comment here
+        valnodes = self.get_nodes("value", root=node)
+        for valnode in valnodes:
+            for att in valnode.attributes:
+                if att.key in attributes:
+                    if re.search(attributes[att.key], att.text):
+                        node.set("value", valnode.text)
```

Comment:
mistake -  It didn't affect us since we aren't creating the case xml files here yet.